peakina/helpers.py

```diff
@@ -81,6 +86,7 @@ class TypeEnum(str, Enum):
     JSON = "json"
     PARQUET = "parquet"
     XML = "xml"
+    GEOJSON = "geojson"
```

Comment:
I don't think we should have `GEOJSON` here. I'd rather have something like `GEODATA`, and let it be a wildcard for everything that can be read by geopandas (even if we only support geojson in laputa for now)